app/src/main/java/org/fossasia/openevent/app/common/utils/core/DateUtils.java

```diff
@@ -67,7 +68,14 @@ public static String formatDateToIso(@NonNull LocalDateTime date) {
     public static ZonedDateTime getDate(@NonNull String isoDateString) {
         if (isoDateString == null)
             return ZonedDateTime.now();
-        return ZonedDateTime.parse(isoDateString).withZoneSameInstant(getZoneId());
+        try {
+            exception = false;
+            return ZonedDateTime.parse(isoDateString).withZoneSameInstant(getZoneId());
+        } catch (DateTimeParseException pe) {
+            Timber.e(pe);
+            exception = true;
```

Comment:
We never ever add a state to pure function, there's a reason why the mehod is static and not in a class